site/source/user-manual/service-guides/vaultwarden/bitwarden-walkthrough.rst

```diff
@@ -0,0 +1,457 @@
+
+        .. figure:: /_static/images/services/vaultwarden/vaultwarden-click-add-folder.png
+                    :width: 50%
+                    :alt: vaultwarden-click-add-folder
+    
+    #. Name the folder and click **Save**:
+
+        .. figure:: /_static/images/services/vaultwarden/vaultwarden-add-folder.png
+                    :width: 50%
+                    :alt: vaultwarden-add-folder
+
+    #. Click **Add item**:
+
+        .. figure:: /_static/images/services/vaultwarden/vaultwarden-add-item.png
+                    :width: 50%
+                    :alt: vaultwarden-add-item
+
+    #. Now add the credentials and click **Save**:
+
+        .. figure:: /_static/images/services/vaultwarden/vaultwarden-add-new-login.png
+                    :width: 50%
+                    :alt: vaultwarden-add-new-login
+
+    #. And there we have it - our first set of login credentials.
+
+        .. figure:: /_static/images/services/vaultwarden/vaultwarden-new-login.png
+                    :width: 50%
+                    :alt: vaultwarden-new-login
+
+    Congratulations! You have setup your own self-hosted password manager and have added a set of login credentials to it!
+
+
+
+    .. _vaultwarden-admin:
+
+.. collapse:: Admin Console
+
+    Accessing the Admin Console
+    ===========================
+    You will find the Admin console by adding ``/admin`` to the end of your Vaultwarden WebUI URL, found in the "Interfaces" section of the Vaultwarden Service page.  You will be prompted for your "Admin Token," which is also found on your Vaultwarden Service page under "Properties."
+
+============
+Client Setup
+============
+
+.. collapse:: Browser Extension
+
+    .. tabs::
+
+        .. group-tab:: Tor
+
+            Browser Extension
+            -----------------
+            If connecting via Tor (i.e using the .onion address) the Bitwarden browser extension will only work with a Tor enabled browser. You can use Firefox (recommended), Tor Browser or Brave Browser.
+
+            #. If you choose Firefox, you will need to :ref:`setup Tor on your device <tor-os>` and :ref:`configure Firefox to use Tor <tor-firefox>`. If using Brave you will just need to :ref:`setup Tor on your device <tor-os>`. With Tor Browser, everything will just work right out of the box.
+
+                .. tip:: We recommend using Firefox as it is the most compatible browser with Start9 Servers.
+
+            #. In this example we will use Firefox, though these instructions will work just the same for Brave. First, install the `Bitwarden browser extension <https://addons.mozilla.org/en-US/firefox/addon/bitwarden-password-manager/>`_.  
+            
+            #. Once installed, click on the extension and click the cog in the top left. Under **Self-hosted environment** you will see a field for **Server URL**. Now we need to grab the address of our Vaultwarden server.
```

Comment:
I think it makes more sense to swap this instruction below the copying step, since the extension disappears every time you click away from it